.pipelines/e2e-with-billing-all-regions.yml

```diff
@@ -0,0 +1,94 @@
+# Azure DevOps Pipeline running RP e2e and Billing e2e in all supported regions
+variables:
+  - template: vars.yml
+stages:
+  - stage: Sign_Off_Stage
+    displayName: Sign Off
+    dependsOn: []
+    jobs:
+    - job: Sign_off_job
+      pool:
+        name: ARO-E2E
+      steps:
+      - script: echo "Kick off E2E"
+  - template: ./templates/template-rp-and-billing-e2e-section.yml
+    parameters:
+      last_stage_of_previous_section: Sign_Off_Stage
+      e2e_subscription: $(e2e-subscription)
+      aro_v4_e2e_devops_spn: $(aro-v4-e2e-devops-spn)
+      billingE2EPipelineName: $(BillingE2EPipelineName)
+      billingE2EBranchName: $(BillingE2EBranchName)
+      gobin: ${{ variables.GOBIN }}
+      gopath: ${{ variables.GOPATH }}
+      goroot: ${{ variables.GOROOT }}
+      modulePath: ${{ variables.modulePath }}
+      locations:
+      - australiaeast
+      - southeastasia
```

Comment:
having southeastasia here doesn't correspond with prod-release.yml - same with uksouth; southafricanorth is missing.
Is there a way to have them all configured in one place?

Reply:
southafricanorth region is having issue with Virtual Network which caused consistent failure, we need address it before adding it back.
Failure msg: "Virtual networks in southafricanorth location do not support ServiceEndpoints for Microsoft.ContainerRegistry service" 
Link: https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=30633641&view=logs&j=e0797121-e35d-5678-4dbf-ddadc7512145&t=2f48ecac-ea49-50d1-2458-aaed9acc1caa

Reply:
prod-release.yml has 4 sectors, given each e2e sector needs over 7 hours run, 4 sectors in e2e means we are not able to finish a single e2e release before the next day's daily e2e kick off.   While running all regions in parallel would also result in resource threshold issue when DeepTest is enabled.  So a trade off is to group regions in 3 sectors for now.